llap-server/src/java/org/apache/hadoop/hive/llap/io/decode/OrcEncodedDataConsumer.java

```diff
@@ -28,16 +28,16 @@
 import org.apache.hadoop.hive.llap.metrics.LlapDaemonQueueMetrics;
 import org.apache.hadoop.hive.ql.exec.vector.ColumnVector;
 import org.apache.hadoop.hive.ql.exec.vector.VectorizedRowBatch;
-import org.apache.hadoop.hive.ql.io.orc.CompressionCodec;
+import org.apache.orc.impl.CompressionCodec;
```

Comment:
The CompressionCodec isn't really an API class, since the user isn't able to add their own implementations (other than the external LZO implementation). I guess I can move it over, if you think it is important. In general LLAP is accessing ORC beneath the API level.